.github/CODEOWNERS

```
@@ -17,6 +17,7 @@ Makefile @webvictim @russjones @awly
 *.md @benarent @webvictim
 /docs/ @benarent @webvictim
 /rfd/ @klizhentas @russjones @benarent
+/content/ @russjones @markherring @benarent @gregchase
```

I'm willing to review this stuff as well if you want.

```suggestion
/content/ @russjones @markherring @benarent @gregchase @webvictim
```